lib/mutant.rb

```diff
@@ -192,7 +193,13 @@ class Config
       reporter:          Reporter::CLI.build($stdout),
       zombie:            false,
       jobs:              Mutant.ci? ? CI_DEFAULT_PROCESSOR_COUNT : ::Parallel.processor_count,
-      expected_coverage: Rational(1)
+      expected_coverage: Rational(1),
+      expression_parser: Expression::Parser.new([
+        Expression::Method,
+        Expression::Methods,
+        Expression::Namespace::Exact,
+        Expression::Namespace::Recursive
```

Comment:
@dkubb It was not part of the PRs description, but this opens the ability to add mutant extensions that need their own expressions such maybe as `erb:logical/name.html.erb`. On bootstrapping / CLI parsing an extension could inject extension specific parsers.